### PR TITLE
Add QFN-28-1EP_4x4mm_P0.4mm_EP2.3x2.3mm

### PR DESCRIPTION
**Pull request by fauxpark**
Required for https://github.com/KiCad/kicad-symbols/pull/1898.

http://www.issi.com/WW/pdf/31FL3731.pdf#page=21

### CODE REVIEW

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-28.yaml [hunk 1]
```diff
@@ -54,6 +54,69 @@ QFN-28-1EP_3x6mm_P0.5mm_EP1.7x4.75mm:
...
+  lead_len:
+    minimum: 0.3
+    nominal: 0.4
+    maximum: 0.5
+
+  EP_size_x:
+    minimum: 2.15
+    nominal: 2.3
+    maximum: 2.4
+  EP_size_x_overwrite: 2.3
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-28.yaml]: unnecessary. nominal is taken by default

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-28.yaml [hunk 2]
```diff
@@ -54,6 +54,69 @@ QFN-28-1EP_3x6mm_P0.5mm_EP1.7x4.75mm:
...
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://www.issi.com/WW/pdf/31FL3731.pdf#page=21'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    nominal: 4
+    tolerance: 0
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-28.yaml]: body size is given as 3.95 .. 4.0 .. 4.05 use the same here.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-28.yaml (hunk 1 above)
Comment (fauxpark) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-28.yaml]: Sorry, just copied the one below. Done.

**Comment by poeschlr** [comment]
@pointhi the footprint is merged, this is good to go.

**Comment by pointhi** [comment]
@poeschlr you show have merge permissions now on this repo :). You can merge changes which only affect the scripts directory now on your own.

**Pull request merged**